docs/pages/admin-guides/deploy-a-cluster/helm-deployments/argocd-helm.mdx

```diff
@@ -0,0 +1,94 @@
+
+This guide leverages ArgoCD's native Helm support to deploy the Teleport agent using the `teleport-kube-agent` Helm chart.
+
+## Prerequisites
+
+- An existing Kubernetes cluster you wish to provide access to via Teleport.
+- (!docs/pages/includes/tctl.mdx!)
+- An existing ArgoCD instance (version 2.10 or greater) that can deploy to the
+  above Kubernetes cluster.
+- The `tsh` client tool v(=teleport.version=)+ installed on your workstation.
+  You can download this from our [installation page](../../../installation/installation.mdx).
+  
+## Step 1/2. Generate a join token
+
+Teleport agents use a join token to obtain certificates and connect to Teleport. See [joining docs](../../../reference/join-methods.mdx) for more information.
+The token is only used to join initially, the Teleport Kube agent will store its
+certificates in Kubernetes and won't need a token to join again in the future.
+In this section, we will create a token for the agent to join the Teleport cluster.
+```code
+$ tctl tokens add --type=kube,app --ttl=5m
+```
+
+You can specify the following token types:
+
+(!docs/pages/includes/token-types.mdx!)
+
+See the `teleport-kube-agent` [chart
+reference](../../../reference/helm-reference/teleport-kube-agent.mdx#roles) for the
+roles and token types that the chart supports.
+
+## Step 2/2. Configure and deploy the `teleport-kube-agent` Helm chart via ArgoCD
+
+1. Create a namespace for Teleport and configure its Pod Security Admission,
+   which enforces security standards on pods in the namespace:
+
+   ```code
+   $ kubectl create namespace teleport
+   namespace/teleport created
+   
+   $ kubectl label namespace teleport 'pod-security.kubernetes.io/enforce=baseline'
+   namespace/teleport labeled
+   ```
+
+2. Create a new ArgoCD application using the following as a template.
+
+``` yaml
+project: default
+source:
+  repoURL: 'https://charts.releases.teleport.dev'
+  targetRevision: (=teleport.version=)
+  helm:
+    values: |-
+      roles: kube,app
+      authToken: $YOUR_AUTH_TOKEN
+      proxyAddr: $YOUR_PROXY_ADDRESS
+      kubeClusterName: $YOUR_KUBE_CLUSTER_NAME
+
+      highAvailability:
+          replicaCount: 2
+          podDisruptionBudget:
+              enabled: true
+              minAvailable: 1
+  chart: teleport-kube-agent
+destination:
+  server: 'https://kubernetes.default.svc'
+  namespace: teleport
+# This section is used to allow the teleport-kube-agent-updater to update the agent
+# without ArgoCD reverting the update.
+ignoreDifferences:
+  - group: apps
+    kind: StatefulSet
+    name: $YOUR_APPLICATION_NAME
+    namespace: teleport
+    jqPathExpressions:
+      - '.spec.template.spec.containers[] | select(.name == "teleport").image'
+```
+
+3. Sync your changes to apply the configuration using `argocd app sync $YOUR_APPLICATION_NAME`
```

Comment:
Can you format the command properly?
```suggestion
3. Sync your changes to apply the configuration using the following command:
```bash
$ argocd app sync $YOUR_APPLICATION_NAME
```
```